### PR TITLE
feat(books): Delete and edit book

### PR DESCRIPTION
**Pull request by emisansev**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (jorge-sanz) [inline npm-debug.log]: This file shouldn't be here

File: src/config.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export const JWT_SECRET = ''; // Random string for JWT Secret. For example: 'whatever-you-want'
-export const DB = ''; // Your DB local direction. For example: 'postgres://yourusername:yourpassword@localhost:5432/halp'
+export const JWT_SECRET = 'secret'; // Random string for JWT Secret. For example: 'whatever-you-want'
```
Comment (jorge-sanz) [inline src/config.ts]: Local dev variables shouldn't be committed

**Review by jorge-sanz — CHANGES_REQUESTED** [review]

File: npm-debug.log (hunk 1 above)
Comment (emisansev) [reply npm-debug.log]: you are right. I was in a hurry, sorry

File: src/config.ts (hunk 2 above)
Comment (emisansev) [reply src/config.ts]: I uploaded the wrong file by mistake. Sorry

**Review by jorge-sanz — APPROVED** [review]

**Pull request merged**